nuxeo-coldstorage-web/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nuxeo-coldstorage-web",
+  "version": "0.1.0",
+  "description": "Nuxeo Cold Storage WebUI integration",
+  "scripts": {
+    "build": "rollup -c"
+  },
+  "author": "Nuxeo",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@rollup/plugin-node-resolve": "^7.1.3",
+    "rollup": "^2.8.0",
+    "rollup-plugin-copy": "^3.3.0",
+    "rollup-plugin-minify-html-literals": "^1.2.5",
+    "rollup-plugin-terser": "^7.0.2"
+  },
+  "dependencies": {
+    "@nuxeo/nuxeo-elements": "3.0.0-SNAPSHOT",
```

Comment:
why are we using 3.0.0-SNAPSHOT here for nuxeo-elements and nuxeo-uil-elements while on nuxeo-web-ui we use ~3.0.2-rc.0?

Reply:
Guess @nelsonsilva copied from and old thing 😅 
Already up to date. Thanks!